<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDatabaseManager</class>
 <widget class="QMainWindow" name="CDatabaseManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCS - Управление базой данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="CAnimatedStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="CDatabaseConnectionExtWidget" name="wConnectingWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="gbxDatabases">
          <property name="title">
           <string>Базы данных</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="4">
            <widget class="QListWidget" name="lwDatabases"/>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Создание базы данных</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Имя новой базы данных</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="leNewDBName"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pbnCreateDB">
                <property name="text">
                 <string>Создать</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pbnConnectToDB">
             <property name="text">
              <string>Подключиться</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pbnDeleteDB">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>308</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="gbxTables">
          <property name="title">
           <string>Таблицы</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="2" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>277</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbnClear">
               <property name="text">
                <string>Очистить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbnDelete">
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="pbnCreateTables">
             <property name="toolTip">
              <string>Создание недостающих таблиц и обновление всех триггеров</string>
             </property>
             <property name="text">
              <string>Создать таблицы</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pbnDeleteAll">
             <property name="text">
              <string>Удалить все</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="lDatabase">
             <property name="text">
              <string>База данных:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QListWidget" name="lwTables"/>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pbnClearAll">
             <property name="text">
              <string>Очистить все</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CAnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>canimatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CDatabaseConnectionExtWidget</class>
   <extends>QWidget</extends>
   <header>cdatabaseconnectionextwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
